src/plugins/intel_cpu/tests/functional/single_layer_tests/mvn.cpp

```diff
@@ -433,13 +433,15 @@ const std::vector<ov::Shape> inputShapesStatic_3D = {
 const std::vector<ov::Shape> inputShapesStatic_4D = {
         {1, 7, 3, 5},
         {1, 15, 9, 5},
-        {4, 41, 6, 9}
+        {4, 41, 6, 9},
+        {1, 4 * 16 * 2 + 16 + 3, 2, 2}
```

Comment:
Maybe we can just use a number directly. Decriptions can be added if necessary.

Reply:
comment is used. thanks